be/src/exec/spill/file_block_manager.cpp

```diff
@@ -45,7 +46,7 @@ class FileBlockContainer {
 
     Status close();
 
-    Dir* dir() const { return _dir; }
+    Dir* dir() const { return _dir.get(); }
```

Comment:
> the raw ptr would be a dead ptr after deleter _dir?

We will not hold the returned result for a long time, the current usage is safe. I use raw pointer here because I do not want to construct a shared_ptr.